jwst/tests_nightly/general/framework/test_self.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_bigdata(_bigdata):
```

Comment:
Please document these tests in this PR, as they actually test the new infrastructure.

Reply:
Will do